.github/workflows/main.yml

```diff
@@ -175,17 +184,9 @@ jobs:
       options: -DCMAKE_BUILD_TYPE=Release -DENABLE_FOLLY=off -G "Visual Studio 17 2022"
 
   test-windows:
-    name: Gtests (Sanity)
+    name: Gtests
     needs: build-windows
     uses: ./.github/workflows/test-windows.yml
     with:
       cache-suffix: ${{ needs.build-windows.outputs.cache-suffix }}
       test: sanity
-
-  modin:
-    name: Integration HDK with Modin (Pytest)
-    uses: ./.github/workflows/modin.yml
-
-  pytest:
-    name: PyHDK (Pytest)
```

Comment:
I don't see where the HDK Python tests are being run, now?

Reply:
It's in `.github/workflows/pytest.yml:64` below (next level job). All jobs in main.yml if they don't `needs` any triggered simultaneously on pull request.